primitives/npos-elections/compact/Cargo.toml

```diff
@@ -22,6 +22,7 @@ proc-macro-crate = "1.0.0"
 
 [dev-dependencies]
 parity-scale-codec = "2.0.1"
+scale-info = { version = "0.10.0", default-features = false, features = ["derive"] }
```

Comment:
sidenote usually dev dependecies we import them with default features and run test only with std, see parity scale codec above for example.